tests/ci/run_x509_limbo.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+set -euxo pipefail
+
+source tests/ci/common_posix_setup.sh
+
+# For now we will just verify that the patch applies and our reporting to builds and tests successfully
+# Subsequent follow-up PRs will wire this up into a new CodeBuild project and handle producing and tracking
+# the reports.
+
+SCRATCH_DIR="${SRC_ROOT}/scratch"
```

Comment:
NP: Some of the integration tests put the scratch folder under `SYS_ROOT` instead of `SRC_ROOT`.  For a script that someone might run in their dev environment, I think using `SYS_ROOT` is preferable.